tokenizer/entities.test

```diff
@@ -2,12 +2,24 @@
 
 {"description": "Undefined named entity in attribute value ending in semicolon and whose name starts with a known entity name.",
 "input":"<h a='&noti;'>",
-"output": [["StartTag", "h", {"a": "&noti;"}]]},
+"output": ["ParseError", ["StartTag", "h", {"a": "&noti;"}]]},
```

Comment:
We don't report error in this case due to:

> If the character reference was consumed as part of an attribute (return state is either attribute value (double-quoted) state, attribute value (single-quoted) state or attribute value (unquoted) state), and the last character matched is not a U+003B SEMICOLON character (;), and the next input character is either a U+003D EQUALS SIGN character (=) or an ASCII alphanumeric, then, for historical reasons, switch to the character reference end state.

https://html.spec.whatwg.org/multipage/parsing.html#named-character-reference-state